### PR TITLE
PAL-1012, Smoke test changes

### PR DESCRIPTION
**Pull request by developer01189998819991197253**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@feature/PAL-1012-ExamplesLoggingDependency')_
```
Comment (developer01189998819991197253) [inline Jenkinsfile:17]: Will revert back after code has passed review stage

**Review by kg981167224 — DISMISSED** [review]

**Review by dev-958 — DISMISSED** [review]

File: templates/classpath-jars-pv.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ metadata:
     type: amazonEFS
     {{- end }}
   annotations:
-    "helm.sh/hook": "pre-install"
+    "helm.sh/hook": "pre-install, pre-upgrade"
```
Comment (dev930018) [inline templates/classpath-jars-pv.yaml]: What was the conclusion on this? If we have it pre-upgrade do we also need a hook-delete-policy?

File: charts/create-kafka-queues/templates/job.yaml [hunk 3]
```diff
@@ -39,7 +39,7 @@ spec:
       initContainers:
       - name: init-wait-for-services
         image: "{{.Values.global.repository}}/{{.Values.image.name}}:{{.Values.image.tag}}"
-        command: ["/bin/wait-for-services.sh"]
+        command: ["/bin/bash", "/bin/wait-for-services.sh" ]
```
Comment (dev930018) [inline charts/create-kafka-queues/templates/job.yaml]: Sorry to nitpick
```suggestion
        command: ["/bin/bash", "/bin/wait-for-services.sh"]
```

File: Jenkinsfile [hunk 4]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@feature/PAL-1012-ExamplesLoggingDependency')_
```
Comment (developer01189998819991197253) [inline Jenkinsfile]:
```suggestion
@Library('jenkinsfile-lib')_
```

File: Jenkinsfile [hunk 5]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@PAL-1012-clean-non-namespaced-resources')_
```
Comment (dev930018) [inline Jenkinsfile]:
```suggestion
@Library('jenkinsfile-lib')_
```

**Review by dev930018 — CHANGES_REQUESTED** [review]

**Review by dev930018 — APPROVED** [review]

**Pull request merged**